test/commandLine.test.js

```diff
@@ -8,7 +8,8 @@ var testHelper = require('./testHelper');
 describe('CommandLineManager', function () {
 
   var argv;
-
+  var proxy_host_actual_value = "http://host";
```

Comment:
what happens if the proxy host inputted by the user is:
1. `https://somehost...`
2. `somehost`
3. `//somehost`

Reply:
It is handled in commandLine.js while parsing --proxy-host considering user provides http/https/no protocol. Do we really need to support protocols like file etc ?

Reply:
No. we need not handle those cases. But having specs around the behaviour helps cover cases:
1. when we pass protocol with the hostname
2. when we don't pass protocol with the hostname

Reply:
added tests

Reply:
can be resolved...